boards/xtensa/esp32/doc/esp32.rst

```diff
@@ -121,31 +121,29 @@ the tool.
 Using JTAG
 ==========
 
-In addition to the SoC vendor and SDK, this board also requires a build of
-OpenOCD with patches supporting the SoC.  The source code and build
-instructions can be obtained from GitHub:
+As with much custom hardware, the ESP-32 moudules require patches to
+OpenOCD that are not upstream.  Espressif maintains their own fork of
+the project here.  By convention they put it in the ~/esp next to the
+installations of their toolchain and SDK:
 
 .. code-block:: console
 
+   cd ~/esp
+   
    git clone https://github.com/espressif/openocd-esp32
 
-After building OpenOCD, one should proceed as usual with the setup process.
-Since JTAG adapters may be different, this will most likely require editing
-the ``esp32.cfg`` file that's in the ``openocd-esp32`` directory.  For
-instance, to use a Flyswatter 2 JTAG adapter, one would modify the file so
-that the "source" line would read:
+   cd openocd-esp32
+   ./bootstrap
+   ./configure
+   make
 
-.. code-block:: tcl
+On the ESP-WROVER-KIT board, the JTAG pins are connected internally to
+a USB serial port on the same device as the console.  These boards
+require no external hardware and are debuggable as-is.
```

Comment:
Jumper wires have to connect TMS, TDO, TDI, and TCK.  (UART requires bridging RXD0/TDX0 and TXD/RXD0.)

The pins are conveniently located in the right side of the board.

Might be a good idea to provide a link to the board documentation, where this (and other things) are explained with pictures: https://dl.espressif.com/doc/esp-idf/latest/get-started/get-started-wrover-kit.html